### ISSUE TITLE
env.get() causing Node-Red to crash

### ISSUE THREAD

**Issue by martoq** [node-red/node-red#3393]
### Current Behavior

When attempting to get an Environment variable passed into a subfunction Node-red crashes.

```
2 Feb 12:38:43 - [red] Uncaught Exception:
2 Feb 12:38:43 - [error] TypeError: RED.settings.functionGlobalContext.get is not a function
    at map (/opt/node_modules/node-red-contrib-actionflows/actionflows/actionflows.js:270:51)
    at EventEmitter.runtimeMap (/opt/node_modules/node-red-contrib-actionflows/actionflows/actionflows.js:593:5)
    at EventEmitter.emit (node:events:390:28)
    at start (/opt/node_modules/@node-red/runtime/lib/flows/index.js:383:12)
```

### Expected Behavior

env.get gets the passed variable

### Steps To Reproduce

It's either crashing on a change node pulling the ENV variable or

```
image  = flow.get("image")
sound  = flow.get("sound")
text   = flow.get("text")
title  = env.get("Title")
url    = flow.get("url")
target = env.get("Target")
delay  = env.get("Delay")

msg.delay = delay*1000

msg.payload = 
{
   "data": {
     "title": title,
     "message": text,
     //"target": target,
     "data": {
       "image": image
  //    "url": url
    }
   }
}
return msg;
```

### Example flow

```

```

### Environment

- Node-RED version: 2.2.0 (Addon version 10.4.0)
- Node.js version: v16.13.2
- npm version: n/a
- Platform/OS: Home Assistant OS 7.2  (armv7 / raspberrypi3)
- Browser: Chrome

**Comment by martoq** [comment]
Further Info:

https://discourse.nodered.org/t/problem-with-node-red-contrib-actionflows-since-upgrade-to-2-2-0/57368

**Comment by hardillb** [comment]
Can you supply a link to the actionflow.js file in use as the one on github (https://github.com/Steveorevo/node-red-contrib-actionflows/blob/master/actionflows/actionflows.js) doesn't look to be up to date and match up with the line numbers in the stacktrace.

**Comment by knolleary** [comment]
See https://github.com/Steveorevo/node-red-contrib-actionflows/issues/19 for details.

**Comment by Steve-Mcl** [comment]
@martoq 

The actionflows author has released a fix to the flows library (V2.1.2)

You can update it via the palette.
